src/modules/rppt_tensor_effects_augmentations.cpp

```diff
@@ -1148,6 +1148,131 @@ RppStatus rppt_pixelate_host(RppPtr_t srcPtr,
     return RPP_SUCCESS;
 }
 
+/******************** fog ********************/
+
+RppStatus rppt_fog_host(RppPtr_t srcPtr,
+                        RpptDescPtr srcDescPtr,
+                        RppPtr_t dstPtr,
+                        RpptDescPtr dstDescPtr,
+                        Rpp32f *gammaTensor,
+                        RpptROIPtr roiTensorPtrSrc,
+                        RpptRoiType roiType,
+                        rppHandle_t rppHandle)
+{
+    RppLayoutParams layoutParams = get_layout_params(srcDescPtr->layout, srcDescPtr->c);
+    Rpp32u height = srcDescPtr->h;
+    Rpp32u width = srcDescPtr->w;
+
+    RpptLayout layout = RpptLayout::NCHW;
+    RppLayoutParams srcLayoutParams = get_layout_params(layout, 1);
+
+    RpptDesc fogMaskSrcDesc ; 
```

Comment:
Please remove the extra space before semicolon here
Applicable to all such instances in this PR